### ISSUE TITLE
query web with empty input causes panic

### ISSUE THREAD

**Issue by Dialga** [nushell/nushell#13496]
### Describe the bug

Running `query web` with an empty input causes a panic.

### How to reproduce

`nu -c 'echo|query web'`

### Expected behavior

Error message but without panicking.

### Screenshots

```
$ nu -c 'echo|query web'
thread 'plugin runner (primary)' panicked at crates/nu_plugin_query/src/query_web.rs:414:42:
Error unwrapping scraperselector::parse: UnexpectedSelectorParseError(EmptySelector)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Error:   × Failed to receive response to plugin call from `query`
   ╭─[source:1:6]
 1 │ echo|query web
   ·      ────┬────
   ·          ╰── while waiting for this operation to complete
   ╰────
  help: try restarting the plugin with `plugin use 'query'`
```

### Configuration

| key                | value                                                            |
| ------------------ | ---------------------------------------------------------------- |
| version            | 0.96.1                                                           |
| major              | 0                                                                |
| minor              | 96                                                               |
| patch              | 1                                                                |
| branch             | makepkg                                                          |
| commit_hash        | f7d6c28a001f94f224e459e21df0da2ab5bdc923                         |
| build_os           | linux-x86_64                                                     |
| build_target       | x86_64-unknown-linux-gnu                                         |
| rust_version       | rustc 1.80.0 (051478957 2024-07-21) (Arch Linux rust 1:1.80.0-1) |
| cargo_version      | cargo 1.80.0 (376290515 2024-07-16)                              |
| build_time         | 2024-07-30 10:25:58 +00:00                                       |
| build_rust_channel | release                                                          |
| allocator          | mimalloc                                                         |
| features           | default, sqlite, system-clipboard, trash                         |
| installed_plugins  | formats 0.96.1, gstat 0.96.1, inc 0.96.1, query 0.96.1           |

**Comment by Embers-of-the-Fire** [comment]
Possible related:

https://github.com/nushell/nushell/blob/d2bf82d22b35d77509f71ebefefb3e3e61c3e278/crates/nu_plugin_query/src/query_web.rs#L412

Uses `expect` but not `Result`.

**Comment by Dialga** [comment]
Hi,
This is still an issue?
```
$ nu -v
0.96.1
$ nu -c 'echo|query web'
thread 'plugin runner (primary)' panicked at crates/nu_plugin_query/src/query_web.rs:414:42:
Error unwrapping scraperselector::parse: UnexpectedSelectorParseError(EmptySelector)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Error:   × Failed to receive response to plugin call from `query`
   ╭─[source:1:6]
 1 │ echo|query web
   ·      ────┬────
   ·          ╰── while waiting for this operation to complete
   ╰────
  help: try restarting the plugin with `plugin use 'query'`
```

**Comment by sholderbach** [comment]
The fix will be released in the 0.97.0 release today. We close the issue when the fix is on the main branch
